frontend/src/modules/scaffold/backboneFormsOverrides.js

```diff
@@ -107,6 +107,7 @@ define([
         htmlSupport: {
           // Convert all allow/disallow strings to regexp, as config is json only
           allow: convertStringsToRegExDeep((Origin.constants.ckEditorHtmlSupport && Origin.constants.ckEditorHtmlSupport.allow) || []),
+          allowEmpty: ['span', 'i'],
```

Comment:
Hey @simondate, I'm curious as to why we're allowing the <I> tag here?

Reply:
What do you mean? Do you think we should also expose options here in `config/config.json`?

I guess that will make this release slightly more future proof? As if in the future more attributes are added we can extend it further there.

Reply:
This was just in the example that I copied from CKEditior's website. I know that some other icon packs such as Font Awesome use `<i>`, so perhaps someone else may find it useful.